### PR TITLE
Mof fix

### PR DESCRIPTION
**Pull request by myrlyn**
This is the beginning of work to fix the large number of Unconstructed References in Monsters of Faerun.
They are mostly monster race Abilities that are referenced in the race.lst, but that do not exist.
I am identifying the missing ones, and implementing them.

### CODE REVIEW

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/faiths_and_pantheons/fp_templates.lst [hunk 1]
```diff
@@ -2,4 +2,5 @@
...
 
-Sycophantically Inspired by Mephit Underlings	VISIBLE:NO	PRECLASS:1,Elemental Archon=3	CHOOSE:NUMBER|MIN=1|MAX=3|TITLE=Number of Underlings Fawning	TEMPDESC:One or more of your mephit underlings have inspirationally fawned over you	TEMPBONUS:ANYPC|CHECKS|ALL|%CHOICE	TEMPBONUS:ANYPC|SKILL|ALL|%CHOICE	TEMPBONUS:PC|COMBAT|TOHIT|%CHOICE	TEMPBONUS:PC|COMBAT|DAMAGE.Melee,DAMAGE.Ranged|%CHOICE
+#Sycophantically Inspired by Mephit Underlings	VISIBLE:NO	PRECLASS:1,Elemental Archon=3	CHOOSE:NUMBER|MIN=1|MAX=3|TITLE=Number of Underlings Fawning	TEMPDESC:One or more of your mephit underlings have inspirationally fawned over you	TEMPBONUS:ANYPC|CHECKS|ALL|%CHOICE	TEMPBONUS:ANYPC|SKILL|ALL|%CHOICE	TEMPBONUS:PC|COMBAT|TOHIT|%CHOICE	TEMPBONUS:PC|COMBAT|DAMAGE.Melee,DAMAGE.Ranged|%CHOICE
+Sycophantically Inspired by Mephit Underlings	VISIBLE:NO	PRECLASS:1,Elemental Archon=3	CHOOSE:NUMBER|MIN=1|MAX=3|TITLE=Number of Underlings Fawning	TEMPDESC:One or more of your mephit underlings have inspirationally fawned over you			BONUS:ANYPC|CHECKS|ALL|%CHOICE			TEMPBONUS:ANYPC|SKILL|ALL|%CHOICE	TEMPBONUS:PC|COMBAT|TOHIT|%CHOICE	TEMPBONUS:PC|COMBAT|DAMAGE.Melee,DAMAGE.Ranged|%CHOICE
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/faiths_and_pantheons/fp_templates.lst]: BONUS:ANYPC is incorrect syntax. Also not related to the topic PR of MoF.

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_ability.lst [hunk 2]
```diff
@@ -0,0 +1,88 @@
+Absorb Magic	KEY:Absorb Magic	TYPE:Supernatural.SpecialQuality	CATEGORY:Special Ability			DESC:Gain HP equal to the damage that damaging magic would do, except for fire and cold magic (which is reduced) and which dissipated after a round.  Nondamaging spells gratn 1HP per spell level.  Mind-control spells have no effect whatsoever.  Can drain 1d4 charges from a magical item on touch, and for every other round of contact thereafter.  Can negate the effect of non-charged items for 1d4 rounds by touch.
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_ability.lst]: This entire file is outside of current conventions. Each race granted ability should follow the KEY:RaceName ~ AbilityName convention. This lacks or follows older outdated conventions.

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_classes.lst [hunk 3]
```diff
@@ -136,16 +136,16 @@ CLASS:Deep Dragon	STARTSKILLPTS:6	MODTOSKILLS:YES			CSKILL:Listen|Search|Spot|Hi
 1																		BONUS:VAR|DragonCL|max(DragonAge,(var("BL=Sorcerer")+var("CL=Sorcerer")))	DEFINE:DragonCL|0
 1	VISION:Blindsense (60')|Darkvision (120')|Keen Senses	BONUS:COMBAT|AC|CL-1|TYPE=NaturalArmor.REPLACE
 ###Block: Breath Weapon and standard immunities
-1	DEFINE:BreathWeaponCone|max((SIZE-1)*10,15)	DEFINE:BreathWeaponDice|(2*DragonAge)	DEFINE:BreathWeaponDamageDie|6	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Cone of Flesh-Corrosive Gas
+1	DEFINE:BreathWeaponCone|0	BONUS:VAR|BreathWeaponCone|max((SIZE-1)*10,15)	DEFINE:BreathWeaponDice|0	BONUS:VAR|BreathWeaponDice|(2*DragonAge)	DEFINE:BreathWeaponDamageDie|0	BONUS:VAR|BreathWeaponDamageDie|6	ABILITY:Special Ability|AUTOMATIC|Breath Weapon ~ Cone of Flesh-Corrosive Gas
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_classes.lst]: While this is correct, the current convention is to move this all out of the class into a proper ability.

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_equip.lst [hunk 4]
```diff
@@ -4,5 +4,8 @@ SOURCELONG:Monster Compendium: Monsters of Faerun	SOURCESHORT:mon	SOURCEDATE:200
...
+Silverweave Armor	PROFICIENCY:ARMOR|Chainmail		TYPE:Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Silverweave	COST:50	WT:20		ACCHECK:-1		EQMOD:STEEL	MAXDEX:5		SOURCEPAGE:p.	SPELLFAILURE:15		BONUS:COMBAT|AC|3|TYPE=Armor
+#Pearl Armor		PROFICIENCY:ARMOR|Pearl Armor		TYPE:Armor.Medium.ArmorProfMedium.ArmorType_Medium.Suit.Silverweave	COST:300	WT:40		ACCHECK:-5		EQMOD:STEEL	MAXDEX:2		SOURCEPAGE:p.	SPELLFAILURE:30		BONUS:COMBAT|AC|5|TYPE=Armor
+#Axe, Throwing			KEY:Axe (Throwing)			PROFICIENCY:WEAPON|Axe (Throwing)			TYPE:Weapon.Container.Melee.Martial Ranged.Finesseable.Ranged.Thrown.Martial.Standard.Slashing.Axe								CONTAINS:UNLIM|Weapon Crystal=1											COST:8	WT:2							CRITMULT:x2	CRITRANGE:1							DAMAGE:1d6				EQMOD:STEEL						WIELD:Light						RANGE:10						SIZE:M	SOURCEPAGE:p.114																																																			SPROP:A throwing axe is lighter than a handaxe and balanced for throwing. Gnome fighters often use throwing axes for both melee and ranged attacks.
+#Gouger Tongue	KEY:Gouger Tongue								TYPE:Natural.Piercing.Finessable.Slashing.Simple		CRITMULT:x2	CRITRANGE:1							DAMAGE:1d8	SIZE:S
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_equip.lst]: Adding commented out items does not make sense to me. Best to make the correction and remove the old item. In git, we can easily track changes. Also decreases the amount of items to review. Currently you have three Pearl Armor changes that I see - the removed, the commented out, and the new. Only having to compare old and new would be better.

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_races.lst [hunk 5]
```diff
@@ -4,7 +4,7 @@ SOURCELONG:Monster Compendium: Monsters of Faerun	SOURCESHORT:mon	SOURCEDATE:200
...
-Artic Dwarf		KEY:Dwarf (Artic)					FAVCLASS:Fighter|Ranger	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:S	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision (60')				LANGBONUS:Giant,Gnome,Goblin,Orc,Terran,Undercommon	BONUS:STAT|STR|4	BONUS:STAT|DEX|-2	BONUS:STAT|CON|2	BONUS:STAT|CHA|-2									AUTO:LANG|Common|Dwarven		ABILITY:Internal|AUTOMATIC|Dwarf Artic Racial Trait|Racial Traits ~ Dwarf	LEGS:2	HANDS:2																					RACETYPE:Humanoid	RACESUBTYPE:Dwarf								TYPE:PC	LEVELADJUSTMENT:2	CR:1/2		SOURCEPAGE:p.	SORTKEY:Dwarf
+Artic Dwarf		KEY:Dwarf (Artic)					FAVCLASS:Fighter|Ranger	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:S	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision (60')				LANGBONUS:Giant,Gnome,Goblin,Orc,Terran,Undercommon	BONUS:STAT|STR|4	BONUS:STAT|DEX|-2	BONUS:STAT|CON|2	BONUS:STAT|CHA|-2									AUTO:LANG|Common|Dwarven		ABILITY:Internal|AUTOMATIC|Dwarf Arctic Racial Trait|Racial Traits ~ Dwarf	LEGS:2	HANDS:2																					RACETYPE:Humanoid	RACESUBTYPE:Dwarf								TYPE:PC	LEVELADJUSTMENT:2	CR:1/2		SOURCEPAGE:p.	SORTKEY:Dwarf
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_races.lst]: Redundant, and not following Race standard convention. Subraces are chosen by ability.

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_races.lst [hunk 6]
```diff
@@ -34,24 +34,26 @@ Banedead																	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Wa
...
+Bat (Deep/Bonebat)			OUTPUTNAME:Deep Bat, Bonebat								STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,10,Fly,50	FACE:5	REACH:5	VISION:Darkvision (60')							PREALIGN:NE								BONUS:COMBAT|AC|2|TYPE=NaturalArmor																																	BONUS:STAT|STR|2	BONUS:STAT|DEX|2	BONUS:STAT|INT|-4	BONUS:STAT|CHA|4								BONUS:VAR|Maneuverability|3																																								MONCSKILL:Hide|Listen|Move Silently|Spot																																																																					ABILITY:Special Ability|AUTOMATIC|Paralysis ~ Bonebat|See Invisibility	ABILITY:Special Ability|AUTOMATIC|Flight Maneuverability																																LEGS:0	HANDS:2	NATURALATTACKS:Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing,*1,1d6																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_races.lst]: I suspect this is not originally a file I created. Not following current conventions at all...

There should only be one single ability call on each race for easier manipulation - ABILITY:Internal|AUTOMATIC|Race Traits ~ Race Name or something similar.

**Comment by FerretDave** [comment]
Arctic dwarf already exists elsewhere - i use it, but cannot recall where,
pc is disconnected currently

On 24 Apr 2017 06:29, "Bahamut" <notifications@github.com> wrote:

*@BahamutDragon* requested changes on this pull request.

This appears to need a lot more work. Let me know if you are willing to
bring this file up to current standards or not. I can accept the changes as
is, but this bloke would like to see it moving towards correct accepted
standards.
------------------------------

In data/35e/wizards_of_the_coast/campaign_settings/forgotten_
realms/faiths_and_pantheons/fp_templates.lst
<https://github.com/BahamutDragon/pcgen/pull/673#discussion_r112866295>:

> @@ -2,4 +2,5 @@
 Elemental Archon Transcendence			VISIBLE:YES	TYPE:Elemental	RACETYPE:Elemental	
 #Elemental Archon
Transcendence			VISIBLE:YES	RACE:.CLEAR	RACESUBTYPE:.CLEAR	RACE:Elemental		RACESUBTYPE:Native	RACESUBTYPE:Air|PREVAREQ:ElementalArchonAir,1	RACESUBTYPE:Earth|PREVAREQ:ElementalArchonEarth,1	RACESUBTYPE:Fire|PREVAREQ:ElementalArchonFire,1		RACESUBTYPE:Water|PREVAREQ:ElementalArchonWater,1

-Sycophantically Inspired by Mephit
Underlings	VISIBLE:NO	PRECLASS:1,Elemental
Archon=3	CHOOSE:NUMBER|MIN=1|MAX=3|TITLE=Number of Underlings
Fawning	TEMPDESC:One or more of your mephit underlings have
inspirationally fawned over
you	TEMPBONUS:ANYPC|CHECKS|ALL|%CHOICE	TEMPBONUS:ANYPC|SKILL|ALL|%CHOICE	TEMPBONUS:PC|COMBAT|TOHIT|%CHOICE	TEMPBONUS:PC|COMBAT|DAMAGE.Melee,DAMAGE.Ranged|%CHOICE
+#Sycophantically Inspired by Mephit
Underlings	VISIBLE:NO	PRECLASS:1,Elemental
Archon=3	CHOOSE:NUMBER|MIN=1|MAX=3|TITLE=Number of Underlings
Fawning	TEMPDESC:One or more of your mephit underlings have
inspirationally fawned over
you	TEMPBONUS:ANYPC|CHECKS|ALL|%CHOICE	TEMPBONUS:ANYPC|SKILL|ALL|%CHOICE	TEMPBONUS:PC|COMBAT|TOHIT|%CHOICE	TEMPBONUS:PC|COMBAT|DAMAGE.Melee,DAMAGE.Ranged|%CHOICE
+Sycophantically Inspired by Mephit
Underlings	VISIBLE:NO	PRECLASS:1,Elemental
Archon=3	CHOOSE:NUMBER|MIN=1|MAX=3|TITLE=Number of Underlings
Fawning	TEMPDESC:One or more of your mephit underlings have
inspirationally fawned over
you			BONUS:ANYPC|CHECKS|ALL|%CHOICE			TEMPBONUS:ANYPC|SKILL|ALL|%CHOICE	TEMPBONUS:PC|COMBAT|TOHIT|%CHOICE	TEMPBONUS:PC|COMBAT|DAMAGE.Melee,DAMAGE.Ranged|%CHOICE

BONUS:ANYPC is incorrect syntax. Also not related to the topic PR of MoF.
------------------------------

In data/35e/wizards_of_the_coast/campaign_settings/forgotten_
realms/monsters_of_faerun/mon_ability.lst
<https://github.com/BahamutDragon/pcgen/pull/673#discussion_r112866407>:

> @@ -0,0 +1,88 @@
+Absorb Magic	KEY:Absorb
Magic	TYPE:Supernatural.SpecialQuality	CATEGORY:Special
Ability			DESC:Gain HP equal to the damage that damaging magic would
do, except for fire and cold magic (which is reduced) and which
dissipated after a round.  Nondamaging spells gratn 1HP per spell
level.  Mind-control spells have no effect whatsoever.  Can drain 1d4
charges from a magical item on touch, and for every other round of
contact thereafter.  Can negate the effect of non-charged items for
1d4 rounds by touch.

This entire file is outside of current conventions. Each race granted
ability should follow the KEY:RaceName ~ AbilityName convention. This lacks
or follows older outdated conventions.
------------------------------

In data/35e/wizards_of_the_coast/campaign_settings/forgotten_
realms/monsters_of_faerun/mon_classes.lst
<https://github.com/BahamutDragon/pcgen/pull/673#discussion_r112866493>:

> @@ -136,16 +136,16 @@ CLASS:Deep Dragon	STARTSKILLPTS:6	MODTOSKILLS:YES			CSKILL:Listen|Search|Spot|Hi
 1																		BONUS:VAR|DragonCL|max(DragonAge,(var("BL=Sorcerer")+var("CL=Sorcerer")))	DEFINE:DragonCL|0
 1	VISION:Blindsense (60')|Darkvision (120')|Keen
Senses	BONUS:COMBAT|AC|CL-1|TYPE=NaturalArmor.REPLACE
 ###Block: Breath Weapon and standard immunities
-1	DEFINE:BreathWeaponCone|max((SIZE-1)*10,15)	DEFINE:BreathWeaponDice|(2*DragonAge)	DEFINE:BreathWeaponDamageDie|6	ABILITY:Special
Ability|AUTOMATIC|Breath Weapon ~ Cone of Flesh-Corrosive Gas
+1	DEFINE:BreathWeaponCone|0	BONUS:VAR|BreathWeaponCone|max((SIZE-1)*10,15)	DEFINE:BreathWeaponDice|0	BONUS:VAR|BreathWeaponDice|(2*DragonAge)	DEFINE:BreathWeaponDamageDie|0	BONUS:VAR|BreathWeaponDamageDie|6	ABILITY:Special
Ability|AUTOMATIC|Breath Weapon ~ Cone of Flesh-Corrosive Gas

While this is correct, the current convention is to move this all out of
the class into a proper ability.
------------------------------

In data/35e/wizards_of_the_coast/campaign_settings/forgotten_
realms/monsters_of_faerun/mon_equip.lst
<https://github.com/BahamutDragon/pcgen/pull/673#discussion_r112866749>:

> @@ -4,5 +4,8 @@ SOURCELONG:Monster Compendium: Monsters of Faerun	SOURCESHORT:mon	SOURCEDATE:200
 # Equipment Name	Required Proficiency			Type									Cost		Weight	AC
Penalty Check	Maximum DEX Bonus	Source Page		% of Spell Failure	Combat
bonus
 #Light Armor
 # Equipment Name	Required Armor
Proficiency		Type						Cost		Weight	AC Penalty Check	Modifier	Maximum
DEX Bonus	Source Page		% of Spell Failure	Combat bonus
-Silverweave Armor	PROFICIENCY:ARMOR|Silverweave
Armor	TYPE:Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Silverweave	COST:50	WT:20		ACCHECK:-1		EQMOD:STEEL	MAXDEX:5		SOURCEPAGE:p.	SPELLFAILURE:15		BONUS:COMBAT|AC|3|TYPE=Armor
-Pearl Armor		PROFICIENCY:ARMOR|Pearl
Armor		TYPE:Armor.Medium.ArmorProfMedium.ArmorType_Medium.Suit.Silverweave	COST:300	WT:40		ACCHECK:-5		EQMOD:STEEL	MAXDEX:2		SOURCEPAGE:p.	SPELLFAILURE:30		BONUS:COMBAT|AC|5|TYPE=Armor
+Silverweave Armor	PROFICIENCY:ARMOR|Chainmail		TYPE:Armor.Light.ArmorProfLight.ArmorType_Light.Suit.Silverweave	COST:50	WT:20		ACCHECK:-1		EQMOD:STEEL	MAXDEX:5		SOURCEPAGE:p.	SPELLFAILURE:15		BONUS:COMBAT|AC|3|TYPE=Armor
+#Pearl Armor		PROFICIENCY:ARMOR|Pearl
Armor		TYPE:Armor.Medium.ArmorProfMedium.ArmorType_Medium.Suit.Silverweave	COST:300	WT:40		ACCHECK:-5		EQMOD:STEEL	MAXDEX:2		SOURCEPAGE:p.	SPELLFAILURE:30		BONUS:COMBAT|AC|5|TYPE=Armor
+#Axe, Throwing			KEY:Axe (Throwing)			PROFICIENCY:WEAPON|Axe
(Throwing)			TYPE:Weapon.Container.Melee.Martial
Ranged.Finesseable.Ranged.Thrown.Martial.Standard.Slashing.Axe								CONTAINS:UNLIM|Weapon
Crystal=1											COST:8	WT:2							CRITMULT:x2	CRITRANGE:1							DAMAGE:1d6				EQMOD:STEEL						WIELD:Light						RANGE:10						SIZE:M	SOURCEPAGE:p.114																																																			SPROP:A
throwing axe is lighter than a handaxe and balanced for throwing.
Gnome fighters often use throwing axes for both melee and ranged
attacks.
+#Gouger Tongue	KEY:Gouger
Tongue								TYPE:Natural.Piercing.Finessable.Slashing.Simple		CRITMULT:x2	CRITRANGE:1							DAMAGE:1d8	SIZE:S

Adding commented out items does not make sense to me. Best to make the
correction and remove the old item. In git, we can easily track changes.
Also decreases the amount of items to review. Currently you have three
Pearl Armor changes that I see - the removed, the commented out, and the
new. Only having to compare old and new would be better.
------------------------------

In data/35e/wizards_of_the_coast/campaign_settings/forgotten_
realms/monsters_of_faerun/mon_races.lst
<https://github.com/BahamutDragon/pcgen/pull/673#discussion_r112866813>:

> @@ -4,7 +4,7 @@ SOURCELONG:Monster Compendium: Monsters of Faerun	SOURCESHORT:mon	SOURCEDATE:200
 ###Block: Base
 #Race Name													Favored Class		Starting
Feats	Size		Move			Face/Space	Reach		Vision						Bonus
Languages							Stat bonus											Racial HD Class
Skills					Automatically Gained Language		Ability												Nb
Legs	Nb Hands	Natural Attacks											Monster Class Name and
Starting Level	Main Race Type	Race Subtype								Type		Level
Adjustment	Challenge Rating	Source Page
 # Race Name		Unique Key						Favored Class		Starting
Feats	Size		Move			Face/Space	Reach		Vision						Bonus
Languages							Stat bonus											Racial HD Class Skills					Auto
Language				Ability												Nb Legs	Nb Hands	Natural
Attacks											Monster Class Name and Starting Level	Main Race
Type	Race Subtype								Type		Level Adjustment	Challenge
Rating	Source Page		SORTKEY
-Artic Dwarf		KEY:Dwarf
(Artic)					FAVCLASS:Fighter|Ranger	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:S	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision
(60')				LANGBONUS:Giant,Gnome,Goblin,Orc,Terran,Undercommon	BONUS:STAT|STR|4	BONUS:STAT|DEX|-2	BONUS:STAT|CON|2	BONUS:STAT|CHA|-2									AUTO:LANG|Common|Dwarven		ABILITY:Internal|AUTOMATIC|Dwarf
Artic Racial Trait|Racial Traits ~
Dwarf	LEGS:2	HANDS:2																					RACETYPE:Humanoid	RACESUBTYPE:Dwarf								TYPE:PC	LEVELADJUSTMENT:2	CR:1/2		SOURCEPAGE:p.	SORTKEY:Dwarf
+Artic Dwarf		KEY:Dwarf
(Artic)					FAVCLASS:Fighter|Ranger	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:S	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision
(60')				LANGBONUS:Giant,Gnome,Goblin,Orc,Terran,Undercommon	BONUS:STAT|STR|4	BONUS:STAT|DEX|-2	BONUS:STAT|CON|2	BONUS:STAT|CHA|-2									AUTO:LANG|Common|Dwarven		ABILITY:Internal|AUTOMATIC|Dwarf
Arctic Racial Trait|Racial Traits ~
Dwarf	LEGS:2	HANDS:2																					RACETYPE:Humanoid	RACESUBTYPE:Dwarf								TYPE:PC	LEVELADJUSTMENT:2	CR:1/2		SOURCEPAGE:p.	SORTKEY:Dwarf

Redundant, and not following Race standard convention. Subraces are chosen
by ability.
------------------------------

In data/35e/wizards_of_the_coast/campaign_settings/forgotten_
realms/monsters_of_faerun/mon_races.lst
<https://github.com/BahamutDragon/pcgen/pull/673#discussion_r112866919>:

> @@ -34,24 +34,26 @@ Banedead																	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Wa
 Baneguard																	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,30		FACE:5	REACH:5	VISION:Darkvision
(60')							PREALIGN:NE								BONUS:COMBAT|AC|3|TYPE=NaturalArmor																																	BONUS:STAT|STR|2	BONUS:STAT|INT|-2	BONUS:STAT|WIS|2	BONUS:STAT|CHA|2																																																				MONCSKILL:Hide|Listen|Move
Silently|Search|Spot																																																												AUTO:WEAPONPROF|Longsword				ABILITY:Special
Ability|AUTOMATIC|Blink (Sp)|Magic Missile (Sp)	ABILITY:Special
Ability|AUTOMATIC|Immunity to
Cold																																	LEGS:2	HANDS:2	NATURALATTACKS:Claw,Weapon.Natural.Melee.Finesseable.Piercing.Slashing,*2,1d4																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.
 Baneguard (Direguard)			OUTPUTNAME:Baneguard,
Direguard							STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,30		FACE:5	REACH:5	VISION:Darkvision
(60')							PREALIGN:NE								BONUS:COMBAT|AC|3|TYPE=NaturalArmor	BONUS:COMBAT|AC|1|TYPE=Deflection																											BONUS:STAT|STR|2	BONUS:STAT|INT|-2	BONUS:STAT|WIS|2	BONUS:STAT|CHA|2																																																				MONCSKILL:Hide|Listen|Move
Silently|Search|Spot																																																												AUTO:WEAPONPROF|Longsword				ABILITY:Special
Ability|AUTOMATIC|Blink (Sp)|Magic Missile (Sp)|See
Invisibility	ABILITY:Special Ability|AUTOMATIC|Immunity to
Cold																														LEGS:2	HANDS:2	NATURALATTACKS:Claw,Weapon.Natural.Melee.Finesseable.Piercing.Slashing,*2,1d4																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.
 Banelar																	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:H	MOVE:Walk,30,Swim,30	FACE:15	REACH:15	VISION:Darkvision
(60')|Low-Light
Vision													BONUS:COMBAT|AC|5|TYPE=NaturalArmor																																	BONUS:STAT|STR|16	BONUS:STAT|DEX|2	BONUS:STAT|CON|10	BONUS:STAT|INT|6	BONUS:STAT|WIS|6	BONUS:STAT|CHA|6		BONUS:VAR|PetrificationResistanceBonus|3	BONUS:VAR|FastHealing|2																																		MONCSKILL:Concentration|Diplomacy|Intimidate|Spot|Use
Magic Device																																					AUTO:LANG|Common|Draconic|Orc																							ABILITY:Special
Ability|AUTOMATIC|Fast Healing|Immunity to Acid|Immunity to
Poison|Poison|Resistance to
Petrification																																	LEGS:0	HANDS:0	NATURALATTACKS:Sting,Weapon.Natural.Melee.Finesseable.Piercing.Poison,*1,2d4|Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing.Poison,*1,1d3																						MONSTERCLASS:Banelar:7				RACETYPE:Magical
Beast							TYPE:Magical Beast	TEMPLATE:Magical
Beast									HITDICEADVANCEMENT:7,15,21					CR:5			SOURCEPAGE:p.
-Bat (Deep/Bonebat)			OUTPUTNAME:Deep Bat,
Bonebat								STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,10,Fly,50	FACE:5	REACH:5	VISION:Darkvision
(60')							PREALIGN:NE								BONUS:COMBAT|AC|2|TYPE=NaturalArmor																																	BONUS:STAT|STR|2	BONUS:STAT|DEX|2	BONUS:STAT|INT|-4	BONUS:STAT|CHA|4								BONUS:VAR|Maneuverability|3																																								MONCSKILL:Hide|Listen|Move
Silently|Spot																																																																					ABILITY:Special
Ability|AUTOMATIC|Paralysis (Su)|See Invisibility	ABILITY:Special
Ability|AUTOMATIC|Flight
Maneuverability																																LEGS:0	HANDS:2	NATURALATTACKS:Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing,*1,1d6																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.
+Bat (Deep/Bonebat)			OUTPUTNAME:Deep Bat,
Bonebat								STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:M	MOVE:Walk,10,Fly,50	FACE:5	REACH:5	VISION:Darkvision
(60')							PREALIGN:NE								BONUS:COMBAT|AC|2|TYPE=NaturalArmor																																	BONUS:STAT|STR|2	BONUS:STAT|DEX|2	BONUS:STAT|INT|-4	BONUS:STAT|CHA|4								BONUS:VAR|Maneuverability|3																																								MONCSKILL:Hide|Listen|Move
Silently|Spot																																																																					ABILITY:Special
Ability|AUTOMATIC|Paralysis ~ Bonebat|See Invisibility	ABILITY:Special
Ability|AUTOMATIC|Flight
Maneuverability																																LEGS:0	HANDS:2	NATURALATTACKS:Bite,Weapon.Natural.Finesseable.Melee.Piercing.Slashing,*1,1d6																													DR:5/Bludgeoning			MONSTERCLASS:Undead:4				RACETYPE:Undead								TYPE:Undead			TEMPLATE:Undead																			CR:2			SOURCEPAGE:p.

I suspect this is not originally a file I created. Not following current
conventions at all...

There should only be one single ability call on each race for easier
manipulation - ABILITY:Internal|AUTOMATIC|Race Traits ~ Race Name or
something similar.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/BahamutDragon/pcgen/pull/673#pullrequestreview-34211766>,
or mute the thread

**Comment by myrlyn** [comment]
I can try rewriting this to match current conventions.

**Pull request closed without merging**